types/react-redux-toastr/index.d.ts

```diff
@@ -30,6 +30,7 @@ interface BasicToastrOptions {
     timeOut?: number;
     transitionIn?: transitionInType;
     transitionOut?: transitionOutType;
+    getState?:(state: ToastrState)=>ToastrState;
```

Comment:
The whitespace here doesn't look like it matches the rest of the file.

Reply:
Thanks.  I have gotten too used to my Prettier plugin fixing things like this on commit.  I updated it.  Please review.